hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -303,6 +305,41 @@ def _apply_updates_impl(
 
         post_apply_fn(updated)
 
+    def get_new_pending_opinion_change(
+        self, metadata: PDQSignalMetadata, new_tags: t.List[str]
+    ):
+        # Figure out if we have a new opinion about this indicator and clear out a pending change if so
+
+        # Threat Updates guarentees there is either 0 or 1 opinion tags on a descriptor
```

Comment:
nit: python-threatexchange/threatexchange/descriptor.ThreatDescriptor.from_te_json is what actually guarantees this